subprojects/docs/src/samples/webApplication/quickstart/build.gradle

```diff
@@ -13,8 +13,17 @@ apply plugin: 'org.gretty'
 // END SNIPPET use-gretty-plugin
 
 // START SNIPPET use-war-plugin
-apply plugin: 'war'
+// START SNIPPET use-gretty-plugin
+plugins {
+// END SNIPPET use-gretty-plugin
+    id 'war'
 // END SNIPPET use-war-plugin
+// START SNIPPET use-gretty-plugin
+    id "org.gretty" version "2.2.0"
+// START SNIPPET use-war-plugin
+}
+// END SNIPPET use-war-plugin
+// END SNIPPET use-gretty-plugin
```

Comment:
Where is this used?

Reply:
_subprojects/docs/src/docs/userguide/webProjectTutorial.adoc_ it seems. That chapter will probably be moved to a Guide at some point.